cmd/util.go

```diff
@@ -313,6 +317,7 @@ func getKubeVersionFromKubectl() *KubeVersion {
 	cmd := exec.Command("kubectl", "version", "-o", "json")
 	out, err := cmd.CombinedOutput()
 	if err != nil {
+		glog.V(2).Info("Failed to query kubectl")
```

Comment:
```suggestion
		glog.V(2).Infof("Failed to query kubectl: %s", err)
```

These errors might say something useful, right?